README.md

```diff
@@ -68,7 +74,7 @@ wget https://download.01.org/intel-sgx/sgx-dcap/1.7/linux/distro/ubuntu18.04-ser
 chmod +x sgx_linux_x64_driver.bin
 sudo ./sgx_linux_x64_driver.bin
 ```
-## Add Intel and Microsoft's repositories & install the required packages
+### Add Intel and Microsoft's repositories & install the required packages
 
 ```bash
 echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
```

Comment:
@rs-- for Ubuntu 20.04, you have to configure this with focal instead of bionic. See https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md for similar instructions